pkg/storage/export/export/export.go

```diff
@@ -811,7 +811,7 @@ func (ctrl *VMExportController) createServiceManifest(vmExport *exportv1.Virtual
 				},
 			},
 			Selector: map[string]string{
-				exportServiceLabel: vmExport.Name,
+				exportServiceLabel: ctrl.getExportPodName(vmExport),
```

Comment:
It does the same thing but I think you should use: ctrl.getExportServiceName(vmExport)

Reply:
I’ll fix this next week and ping you back when done. Thanks.

Reply:
@ShellyKa13 All done. Both lines should now be using `getExportServiceName` instead of `getExportPodName`.